pkg/cli/cmd/install/kubernetes/kubernetes.go

```diff
@@ -53,6 +53,9 @@ rad install kubernetes --kubecontext mycluster
 
 # Install Radius with overrides in the current Kubernetes context
 rad install kubernetes --set key=value
+
+# Install Radius with the intermediate root CA certificate in the current Kubernetes context
+rad install kubernetes --set-file global.rootCA.cert=/path/to/rootCA.crt
```

Comment:
@rynowak helm chart doesn't allow chart to load external files via `.File.Get` due to the security issue. Instead, it has `--set-file` flag to let `helm cli` serialize files and pass it as a string. So I introduce the same param in `rad install kubernetes` in order to create root ca secret. Please share your thought to introduce new `--set-file` flag.

Reply:
I'm in favor of this.